cmd/keysync/keysync.go

```diff
@@ -74,24 +74,26 @@ func main() {
 		}
 	}
 
-	raven.CapturePanicAndWait(func() {
-		metricsHandle := sqmetrics.NewMetrics("", config.MetricsPrefix, http.DefaultClient, 1*time.Second, metrics.DefaultRegistry, &stdlog.Logger{})
+	//raven.CapturePanicAndWait(func() {
```

Comment:
Why a comment here?

Reply:
Whoops, that's a mistake

Reply:
Fixed, and now re-panicing.  I had commented this out so I could see a panic in dev without sentry.